### PR TITLE
Feat: Add Upwork Feed page/table

### PR DESCRIPTION
**Pull request by Max-Shulga**
- Implemented sorting functionality by date, title, keywords, and scores.
- Customized item display count per page.
- Added pagination to navigate through pages.
- Adjusted page layout to adhere to design specifications.

### CODE REVIEW

File: src/styles/react-select.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+.custom-menu.custom-menu{
+    border-radius: 0.75rem;
+    border: 1px solid;
+    z-index: 11;
+}
+
+.custom-menu-light.custom-menu-light{
+    background: #FFFFFF;
+    border-color:  #D5D7DB;
```
Comment (AndriiKasian) [inline src/styles/react-select.css:52]: - [ ] move reusable colors to variables, check all entries

File: src/theme/theme.ts [hunk 2]
```diff
@@ -11,6 +11,13 @@ export const getDesignTokens = (mode: PaletteMode): ThemeOptions => ({
           text: {
             primary: "#252733",
           },
+          score: {
+            red: "#FAC8D0",
```
Comment (AndriiKasian) [inline src/theme/theme.ts:15]: - [ ] declare variables for colors and update them in all files where you use colors

**Review by AndriiKasian — CHANGES_REQUESTED** [review]

File: src/theme/theme.ts (hunk 2 above)
Comment (Max-Shulga) [reply src/theme/theme.ts]: done

File: src/styles/react-select.css (hunk 1 above)
Comment (Max-Shulga) [reply src/styles/react-select.css]: done

**Review by AndriiKasian — APPROVED** [review]

**Pull request merged**